console/src/schematic/toolbar/Toolbar.tsx

```diff
@@ -102,7 +105,16 @@ export const Toolbar = ({ layoutKey }: ToolbarProps): ReactElement | null => {
     >
       <ToolbarHeader>
         <ToolbarTitle icon={<Icon.Schematic />}>{name}</ToolbarTitle>
-        <Tabs.Selector style={{ borderBottom: "none" }} />
+        <Align.Space direction="x" align="center" size={0}>
```

Comment:
Some comments on the UI:

Adding this extra button forces the "Properties" Tab to go off the page slightly, making things look strange.

There should be some breathing room between the export button and the tab selectors. Otherwise it makes it seem like the tabs and the button are elements related in functionality, which they're not. Add a divider and some space. Let it breathe!

The rounded corners on the button also look strange as the shadow of the button hits the edges of the toolbar header. Either make the button smaller so it sits well inside the toolbar, or make the edge border radius zero by using the `sharp` prop on the button.